esm/shared/utils.js

```diff
@@ -38,3 +38,14 @@ export const setAdjacent = (prev, next) => {
   if (next)
     next[PREV] = prev;
 };
+
+export const htmlToFragment = (ownerDocument, html) => {
+  const fragment = ownerDocument.createDocumentFragment();
+
+  const elem = ownerDocument.createElement('');
+  elem.innerHTML = html;
+
+  for (const node of elem.childNodes) fragment.appendChild(node.cloneNode(true));
```

Comment:
why `cloneNode(true)` is needed in here? It's also worth mentioning this is a classic *move nodes* operations which is *O(1)* operation changing the `PREV` and `NEXT` links to the first and last child + updating `PARENT` ... mind refactoring this to make it faster? If needed some hint around *how to* check what `appendChild` does internally